pyinjective/core/network.py

```diff
@@ -83,7 +86,12 @@ def _is_cookie_expired(self, cookie_data: str) -> bool:
         cookie = SimpleCookie()
         cookie.load(cookie_data)
 
-        expiration_time = datetime.datetime.strptime(cookie["GCLB"]["expires"], "%a, %d-%b-%Y %H:%M:%S %Z").timestamp()
+        if "GCLB" not in cookie:
+            expiration_time = 0
+        else:
+            expiration_time = datetime.datetime.strptime(
+                cookie["GCLB"]["expires"], "%a, %d-%b-%Y %H:%M:%S %Z"
```

Comment:
Not in the examples I checked. If that happens the expiration check should fail, since that would be unexpected.
There is no way to test it correctly now because there are no k8s servers supported